### PR TITLE
Create dummy API of AudienceActions.

### PR DESCRIPTION
**Pull request by hot2346**
Create an entrypoint of the API of RevenueActions.

### CODE REVIEW

File: src/runtime/runtime.js [hunk 1]
```diff
@@ -515,6 +515,13 @@ export class Runtime {
       )
     );
   }
+
+  /** @override */
+  showBestAudienceAction() {
+    console.error('Forbidden call, under implementation.');
```
Comment (hartmannr76) [inline src/runtime/runtime.js:521]: Maybe use `debugLog` for these for now. Linter doesn't seem to like the `console` messaging

File: src/api/subscriptions.js [hunk 2]
```diff
@@ -325,6 +325,15 @@ export class Subscriptions {
...
    */
   consumeShowcaseEntitlementJwt(showcaseEntitlementJwt, onCloseDialog) {}
+
+  /**
+   * Intelligently returns the most interesting action to the
+   * reader based on different different user status. For
+   * instance, a new user may get free metering by simply
+   * clicking 'follow-publisher' action, and a frequently
+   * visiting user may be shown a 'creating an account' action.
+   */
```
Comment (hartmannr76) [inline src/api/subscriptions.js:335]: Include the return type in the comment `* @return {?}`

File: src/runtime/runtime.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/runtime/runtime.js:521]: `warn` (from "src/utils/log.js") might be even better

File: src/api/subscriptions.js (hunk 2 above)
Comment (hot2346) [reply src/api/subscriptions.js:335]: Will fix in next commit.

**Pull request closed without merging**